fern/docs.yml

```diff
@@ -58,6 +58,8 @@ navigation:
         contents:
           - page: Vector Stores
             path: ./docs/pages/manual/vectordb.mdx
+          - page: SQL Databases
+            path: ./docs/pages/manual/nlsql.mdx
```

Comment:
I don't think this should be under `Storage` section, but more into `Advance Setup` or something like this.